cmd/sops/version.go

```diff
@@ -10,7 +10,7 @@ import (
 	"gopkg.in/urfave/cli.v1"
 )
 
-const version = "2.0.9"
+const version = "2.0.10"
```

Comment:
You shouldn't bump the version. It gets painful when other pull requests are merged prior to the one bumping the version, so I do it in the master branch directly, or in a separate PR.